fair_mango/metrics/base.py

```diff
@@ -286,21 +358,48 @@ def __init__(
             self.label1 = "unprivileged"
             self.label2 = "privileged"
         else:
-            raise ValueError(
-                "Metric type not recognized. accepted values 'performance' or 'error'"
+            raise AttributeError(
+                "Metric type not recognized. accepted values 'performance' or "
+                "'error'"
             )
 
         self.result: dict | None = None
         self.ranking: dict | None = None
         self.results: dict | None = None
 
     def _compute(self) -> dict[tuple, np.ndarray[float]]:
+        """Calculate the disparity in the scores between every possible pair in
+        the provided groups.
+
+        Returns
+        -------
+        dict[tuple, np.ndarray[float]]
+            A dictionary with:
+            - keys: tuple with the pair of the sensitive groups labels.
+            - values: a numpy array with the corresponding disparity.
+        """
         metric = self.metric(self.data, **self.kwargs)
         self.targets, self.metric_results = metric()
         results = calculate_disparity(self.metric_results, "difference")
         return results
 
     def summary(self) -> dict[str, dict[str, float | tuple | None]]:
+        """Return the fairness metric value, in other words the biggest
+        disparity found with specifying the priviliged and discriminated
+        groups.
+
+        Returns
+        -------
+        dict[str, dict[str, float | tuple | None]]
+        A dictionaty with:
+        - keys: name of the target variable.
+        - values: a dictionary corresponding to the results for that target
+        variable with:
+            - keys: labels for the biggest disparity, the privileged group
+            and the discriminated group.
```

Comment:
so there are only two keys in your dictionary?

Reply:
No, there are 3 labels:
- The biggest disparity (The name will depend on the fairness metric. Ex: `demographic_parity_difference`, `disparate_inpact_ratio`...).
- `priveleged`: this will hold the privileged group.
- `unprivileged`: this will hold the unprivileged or discriminated group.